lib/car.rb

```diff
@@ -0,0 +1,36 @@
+class Car
+
+  def initialize(make, monthly_payment, loan_length)
+    @make = make
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    @make = "Ford"
+  end
+
+  def model
+    @model = "Mustang"
+  end
+
+  def monthly_payment
+    @monthly_payment = 1500
+  end
+
+  def loan_length
+    @loan_length = 36
+  end
+
+  def total_cost
+    @loan_length.to_f * @monthly_payment.to_f
+  end
+
+  def color
+
+  end
+
+  def paint!
```

Comment:
In the interaction pattern, when we call the `paint!` method we pass it an argument, so you need to write this method so that it accepts an argument: `def paint!(new_color)`